src/client/shadows/dynamicshadowsrender.h

```diff
@@ -76,6 +76,7 @@ class ShadowRenderer
 	void setClearColor(video::SColor ClearColor);
```

Comment:
should be removed if it's no longer used

Reply:
Also, why *is* it no longer needed?

Reply:
Done. 

@lhofhansl It's because ShadowRenderer is no longer reponsible setting up the render target for scene manager. This is now moved back to RenderCore and descendants.